rero_ils/modules/libraries_locations/templates/rero_ils/detailed_view_libraries_locations.html

```diff
@@ -21,8 +21,19 @@ <h3>{{ record.name }}</h3>
     {{ div_row_xs3_xs9(_('Email'), record.email) }}
   {% endif %}
 
+  <!-- Library settings -->
+  <h4>Opening hours and holidays</h4>
+  <div class="row">
+    <div class="col-md-6">
+      {% include('rero_ils/_opening_hours.html') %}
+    </div>
+    <div class="col-md-6">
+      <b>Exceptions (holidays, etc.)</b>
+    </div>
+  </div>
+
   {% with
-    href_update=url_for('lib.edit_view', pid=record.pid),
+    href_update= '/admin/libraries/'+ record.pid,
```

Comment:
`url_for('libraries.index', path=record.pid)` or something like it.

Reply:
Changed